erpnext/crm/doctype/opportunity/opportunity.py

```diff
@@ -540,20 +584,13 @@ def submit_communication(name, contact_date, remarks, submit_follow_up=False):
 
 	opp = frappe.get_cached_doc('Opportunity', name)
```

Comment:
use `frappe.get_doc` instead of frappe.get_cached_doc`

- if we're loading something for the purposes of updating... we don't rely on cache
- if we're loading a transaction ... we don't cache